website/thaliawebsite/templates/index.html

```diff
@@ -26,11 +26,13 @@ <h1 class="text-center section-title">{% trans "Study Association Thalia" %}</h1
 
     {% render_frontpage_articles %}
 
+    {% render_event_cards %}
+
     <section class="page-section">
         <div class="container">
             <div class="row">
                 <div class="col-12 col-md-6">
-                    {% render_frontpage_events %}
```

Comment:
Are you keeping this tag on purpose? What about the alumni page?

Reply:
I figured alumni are not that interested in summaries, so I kept the layout there

Reply:
Summaries? huh?

This exact tag is also used on the alumni page to render the alumni-only events there. I changed that to the list to not keep both, but that is a good place for the cards imo.

Reply:
Nvm I was confused. I don't necessarily want to remove the code because it is a nice building block...